### PR TITLE
Add the option to geo-block IP addresses

### PR DESCRIPTION
**Pull request by pixeltrix**
If there is a persistent attack using botnets from abroad we want to have the option to block ip addresses from outside the UK.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ gem 'lograge'
 gem 'logstash-logger'
 gem 'jbuilder'
 gem 'paperclip'
+gem 'maxminddb'
```
Comment (kalleth) [inline Gemfile]: IIRC maxminddb needs frequent updating to remain current -- are we going to reflect that with a script on the ops side?

File: app/assets/stylesheets/petitions/_forms.scss [hunk 2]
```diff
@@ -87,3 +87,4 @@ select.form-control:hover {
 .form-hint {
   margin-top: 10px;
 }
+
```
Comment (kalleth) [inline app/assets/stylesheets/petitions/_forms.scss]: churn :nitpick:

File: Gemfile (hunk 1 above)
Comment (pixeltrix) [inline Gemfile]: Yes 😉

File: app/assets/stylesheets/petitions/_forms.scss (hunk 2 above)
Comment (pixeltrix) [inline app/assets/stylesheets/petitions/_forms.scss]: fixed 😄

**Comment by kalleth** [comment]
The code here looks good. :+1: 

[Maxminddb has a 99.8% accuracy rating on countries](https://support.maxmind.com/geoip-faq/geoip2-and-geoip-legacy-databases/how-accurate-are-your-geoip2-and-geoip-legacy-databases/), which might mean we prevent (when this feature has been enabled) 0.2% of the UK from signing permissions.

OTOH, 0.2% of valid submissions not getting through vs preventing an extra 100%+ of false submissions is probably a good tradeoff.

**Comment by pixeltrix** [comment]
The geoipupdate changes have been applied to the preview environment so we're good to merge and test this.

**Pull request merged**